tests/test_cli.py

```diff
@@ -432,6 +431,7 @@ def test_echo_undefined_variable_error(output_dir, cli_runner):
             '_template': template_path,
             '_repo_dir': template_path,
             '_output_dir': output_dir,
+            '_checkout': None,
```

Comment:
I’m also willing to write additional use cases if necessary, but because of how these private context keys are managed, they often aren’t available in happy path outlets, so I would just be writing a test case very similar to this one, but with a customized check out being reflected in the context.

Open to other testing approaches. One I considered was creating and checking a replay file